docs/testing/sideload-office-add-ins-for-testing.md

```diff
@@ -65,40 +70,24 @@ This method doesn't use the command line and can be accomplished using commands
 
     ![The upload add-in dialog with buttons for browse, upload, and cancel.](../images/upload-add-in.png)
 
-1. Verify that your add-in is installed. For example, if it is an add-in command, it should appear on either the ribbon or the context menu. If it is a task pane add-in, the pane should appear.
+1. Verify that your add-in is installed. For example, if it has an add-in command, it should appear on either the ribbon or the context menu. If it is a task pane add-in that has no add-in commands, the task pane should appear.
```

Comment:
```suggestion
1. Verify that your add-in is installed. For example, if it has an add-in command, it should appear on either the ribbon or the context menu. If it's a task pane add-in without add-in commands, the task pane should appear.
```

Reply:
"without" is ambiguous, but I fixed the contraction